### ISSUE TITLE
Support `.github/kodiak.toml` (not just `.github/.kodiak.toml`)

### ISSUE THREAD

**Issue by infokiller** [chdsbd/kodiak#494]
Context: I like having my config files centralized to avoid clutter/pollution (similarly to `~/.config` in Linux/XDG). When these files are centralized, it's not necessary to name them with a prefix dot (`.filename`), since they no longer add clutter, and it's actually somewhat better not to name them this way, since then `ls` will show them by default.
With that context in mind, I'd like to suggest supporting `.github/kodiak.toml` as well. This is also more consistent with other files in that directory that don't have a leading dot in their name (`CONTRIBUTING.md`, `ISSUE_TEMPLATE`, `dependabot.yml`, `workflows`, etc).

**Comment by chdsbd** [comment]
Good idea.

This isn't too hard to implement. We need to add a couple more options to the query that fetches the files: https://github.com/chdsbd/kodiak/blob/2965d2e79f0a7b3984ad64cd57b3b082d6144a59/bot/kodiak/queries.py#L196-L205

For consistency we should probably support `kodiak.toml` in the root too. So users could do `.kodiak.toml`,`kodiak.toml`,`.github/.kodiak.toml` and `.github/kodiak.toml`.

This is the PR (https://github.com/chdsbd/kodiak/pull/227) that added support for `.github/.kodiak.toml`. Adding support for these new paths would involve similar changes:

**Comment by infokiller** [comment]
One thing that I guess needs to be sorted out is the priority of these files in case the user provides multiple ones (probably a user error, as I see no reason to do that).

**Comment by chdsbd** [comment]
Yeah, right now we just go in some order: https://github.com/chdsbd/kodiak/blob/2965d2e79f0a7b3984ad64cd57b3b082d6144a59/bot/kodiak/queries.py#L892-L899

We could probably raise a configuration error to the user if multiple configuration files are provided.
